apps/wing-console/console/server/src/router/app.ts

```diff
@@ -601,12 +601,21 @@ export const createAppRouter = () => {
         }),
       )
       .mutation(async ({ ctx, input }) => {
-        ctx.emitter.emit("openFileInEditor", {
-          path: input.path,
-          line: input.line,
-          column: input.column,
-        });
+        if (ctx.emitter.listenerCount("openFileInEditor") > 0) {
+          ctx.emitter.emit("openFileInEditor", {
+            path: input.path,
+            line: input.line,
+            column: input.column,
+          });
+          return;
+        }
+        const launch = require("launch-editor");
+        launch(`${input.path}:${input.line}:${input.column}`);
       }),
+
+    // Subscribing to this procedure will override the default behavior of opening files in the editor
+    // provided by the "app.openFileInEditor" procedure.
+    // Needed to manage file opening within the vs-code extension.
```

Comment:
This should be a `/** ... */` docstring, otherwise it won't show up in the IDE when consuming the subscription.

```suggestion
    /**
     * Warning! Subscribing to this procedure will override the default behavior of opening files in the editor
     * provided by the "app.openFileInEditor" procedure.
     * Needed to manage file opening within the vs-code extension.
     */
```